### ISSUE TITLE
Couldn't find helm binary when installing Testkube components

### ISSUE THREAD

**Issue by mgoodwin1989** [kubeshop/testkube#1309]
**I followed the Testkube plugin installation instruction and there is no mention of helm needing to be installed but when I tried to install the Testkube components using command `kubectl testkube install` I got the following error: 

**To Reproduce**
1. Run `kubectl testkube install`
2. See error 'Installing testkube (error: exec: "helm": executable file not found in $PATH)'**

**Expected behavior**
Teskube components would be installed in the Kubenetes cluster

**Version / Cluster**
- Testkube version - 1.0.3
- Kubernetes cluster - Minikube
- Kubernetes version - 1.25.2

**Comment by bioslikk** [comment]
@kubeshop/testkube Would it be possible to install helm in case it's not installed ? Either Way, probably worth to add helm as a requirement in our docs.

**Comment by vsukhin** [comment]
@bioslikk docs were update there https://github.com/kubeshop/testkube/pull/1310

**Comment by dimonlex** [comment]
Thanks for pointing it out @mgoodwin1989.  Are you all set otherwise?

**Comment by mgoodwin1989** [comment]
Yep. Just getting on board with TestKube. I was able to install components.
I’m good to go. Thanks for checking back with me.

Thanks,

Matt

**Comment by exu** [comment]
I assume that we could use helm api from some kind of helm go library? For sure it would need to be researched but should be possible- we would get rid of helm binary dependency then. @vsukhin @nicufk @vLia what do you think?

**Comment by vsukhin** [comment]
yes, helm 3 has go sdk. so, technically we can have a deep integraion with helm if we want, but it also means that testkube will contain helm inside, even if most of the users will already have helm installed. For me, it's really questionable, depending if users have real demand for this
